corehq/apps/cloudcare/templates/formplayer/users.html

```diff
@@ -24,40 +24,91 @@ <h1 class="page-title">Login as user</h1>
+                </a>
+                <a class="js-user-previous">
+                  <span><i class="fa fa-caret-left"></i></span>
+                </a>
+              </li>
+              <!-- Render page buttons -->
+              <% if (pagesToShow.pageCount <= 5) { %>
+                  <% for (i = pagesToShow.startPage-1; i < pagesToShow.endPage; i++) { %>
+                    <li class="js-page<% if (i+1 === page) { %> active<% } %>" data-page="<%= i+1 %>"><a><%= i + 1 %></a></li>
+                  <% } %>
+              <% } else { %>
+                <% if (pagesToShow.startPage <= 3) { %>
+                  <% for (i = 1; i < pagesToShow.startPage; i++) { %>
+                    <li class="js-page<% if (i === page) { %> active<% } %>" data-page="<%= i %>"><a><%= i %></a></li>
+                  <% } %>
+                <% } else { %>
+                    <li class="js-page<% if (i === page) { %> active<% } %>" data-page="1"><a>{% trans "1" %}</a></li>
+                    <li class="js-page"><a class="paginate-disabled">{% trans "..." %}</a></li>
+                <% } %>
+                <% for (i = pagesToShow.startPage; i <= pagesToShow.endPage; i++) { %>
+                  <li class="js-page<% if (i === page) { %> active<% } %>" data-page="<%= i %>"><a><%= i %></a></li>
+                <% } %>
+                <% if (pagesToShow.endPage >= pagesToShow.pageCount - 2) { %>
+                  <% for (i = pagesToShow.endPage+1; i <= pagesToShow.pageCount; i++) { %>
+                    <li class="js-page<% if (i === page) { %> active<% } %>" data-page="<%= i %>"><a><%= i %></a></li>
+                  <% } %>
+                <% } else { %>
+                  <li class="js-page"><a class="paginate-disabled">{% trans "..." %}</a></li>
+                  <li class="js-page<% if (i === page) { %> active<% } %>" data-page="<%= pagesToShow.pageCount %>"><a><%= pagesToShow.pageCount %></a></li>
+                <% } %>
+              <% } %>
+              <!-- Render Next buttons -->
+              <% if (page === totalPages) { %>
+              <li class="disabled">
+                <% } else { %>
+              <li>
+                <% } %>
+                <a class="js-user-next">
+                  <span><i class="fa fa-caret-right"></i></span>
+                </a>
+                <a class="js-page" data-page="<%= pagesToShow.pageCount %>">
+                  <span><i class="fa fa-fast-forward"></i></span>
+                </a>
+              </li>
+            </ul>
+            </nav>
+          </div>
+          <div class="col-sm-3">
+            <div class="form-inline paginate-goto">
+                <span class="page-text">{% trans "Go to page" %}</span>
+                <input type="text" class="form-control pagination-user-form" style="width: 60px;" placeholder="Page" id="goText">
+                <input type="button" class="btn btn-primary js-page-go-search pagination-user-form" value="Go">
```

Comment:
To be semantic, use `<button>` for buttons instead of `<input type="button"/>`. This one should also be `btn-default`.